snippets/visualbasic/VS_Snippets_Misc/tpl_parallel/vb/simpleforeach.vb

```diff
@@ -1,33 +1,31 @@
 ﻿'<snippet03>
 ' How to: Write a Simple Parallel.ForEach Loop
 ' IMPORTANT!!!: Add reference to System.Drawing.dll
```

Comment:
this can be deleted; it's no longer necessary to add a reference to the assembly.